src/server.h

```diff
@@ -1485,6 +1490,7 @@ typedef struct serverTLSContextConfig {
     char *client_cert_file;     /* Certificate to use as a client; if none, use cert_file */
     char *client_key_file;      /* Private key filename for client_cert_file */
     char *client_key_file_pass; /* Optional password for client_key_file */
+    int client_auth_user;      /* Field to be used for Authomatic TLS Authenticaton */
```

Comment:
```suggestion
    int client_auth_user;      /* Field to be used for automatic TLS authentication based on client TLS certificate */
```
Although, authomatic sounds like a great name for this automatic authentication :)